media/stylus/components/compat-tables/bc-browser.styl

```diff
@@ -0,0 +1,93 @@
+/*
+Icons in mobile view
+====================================================================== */
+
+@media $media-query-small-mobile {
+
+    [class*='bc-browser-']:before,
```

Comment:
Actually, since this is a class, if the name will always start with `bc-browser-` we could even consider using the standard class selector as below.